tox.ini

```diff
@@ -1,3 +1,6 @@
 [flake8]
 exclude = amgut/lib/locale_data/*,amgut/templates/*,amgut/static/*,amgut/db/*
-max-complexity = 10
\ No newline at end of file
+max-complexity = 10
+
+[pep257]
+ignore = D100,D102,D200,D203,D204,D205,D400
```

Comment:
Not sure if we should be ignoring all these, and if we add this section are we going to then start enforcing pep257 on this repo?